pkg/telemetry/telemetry.go

```diff
@@ -268,6 +275,60 @@ func (t *engineImpl) syncInfoTable(stats map[string]float64) error {
 	return nil
 }
 
+func (t *engineImpl) housekeeping() {
+	var lastRunBeyondHour bool
```

Comment:
From the point of view of the Go code it doesn't care whether the run was beyond an hour. It cares that the housekeeper performed the update. So this should be called 
```suggestion
	var housekeeperPerformedUpdate bool
```

Then the logic is if houskeeperPerformedUpdate=>getInstanceInformationStats + writeThem

This decouples the internal time-based logic in SQL from the go code that just cares if there was an update

Reply:
I agree, but just to clear the logic, we need to write getInstanceInformationStats, not because housekeeper performed the update, but because no matter if housekeeper performed update or not, its past 1 hour, so we must write the new information stats, which is why I wrote it. This also means that the SQL function should return true, as soon as its beyond the 1 hour duration, and not only when the counter reset row was updated.